tests/cross/src/test/scala/tests/pc/CompletionOverrideSuite.scala

```diff
@@ -844,7 +1014,13 @@ class CompletionOverrideSuite extends BaseCompletionSuite {
        |overTop: Int
        |""".stripMargin,
     includeDetail = false,
-    topLines = Some(2)
+    topLines = Some(2),
+    compat = Map(
+      "3" ->
+        """|override def overTop: Int
```

Comment:
Yes, if we remove `topLines = Some(2)`, we'll get

```
=> Obtained
    """|override def overTop: Int
       |override def equals(x$0: Any): Boolean
       |override def hashCode: Int
       |override def toString: String
       |overTop: Int
       |""".stripMargin
```

It would be ideal to place basic `overTop` completion on the second, but not sure how Scala2 does 🤔 

I'll remove `topLines = Some(2)` when it's Scala3 so we can test basic `overTop` completion is included here.

Reply:
I think that a fuzzy match on the name should be first and only then we should see the other overrides.

Reply:
Thanks to https://github.com/scalameta/metals/pull/3897#discussion_r875491408 the test result is now much better :tada: